docs/website/docs/reference/tuning.md

```diff
@@ -1,36 +1,47 @@
 ---
+hide:
+  - tags
+tags:
+  - sharktuner
+  - SHARKTUNER
+  - Tuner
 icon: octicons/meter-16
 status: new
 ---
 
 # Tuning
 
-This page documents support for IREE dispatch tuning. The compiler supports
-both default and user-provided tuning specs (specifications) that override
-compiler heuristics that guide dispatch code generation. In our experience,
-tuning specs can provide meaningful speedup of model execution. For example, we
-achieved a ~10% improvement on the Stable Diffusion XL (SDXL) model with the
-MI300X GPU.
+Tuning is an important step in the process of optimizing performance for your
+applications. The IREE compiler provides default tuning specs (specifications)
+but also supports user-provided tuning specs through the use of our dispatch
+tuner, [SHARK Tuner](https://github.com/nod-ai/shark-ai/tree/main/sharktuner),
+which provides one way to generate tuning specs and run them through a tuning
+loop.
 
-## Tuning specs
+!!! tip
+    In our experience, tuning specs can provide **meaningful speedup** of model
+    execution. For example, SHARK Tuner achieved a ~10% improvement on the SDXL
+    (Stable Diffusion XL) model with the MI300X GPU.
 
-The default specs are shipped with the IREE compiler and are target-specific.
-We aim to provide default tuning specs that cover the most in-demand hardware
-and dispatches from most popular ML models, although we do not guarantee
-completeness.
+## :octicons-book-16: Overview
 
-User-provided tuning specs are a mechanism that allows for users to get the
-best performance on custom models and hardware targets without having to modify
-the compiler source code or needlessly special-case compiler heuristics.
+The IREE compiler is shipped with target-specific, default specs. With these
+default specs, we aim to cover the most in-demand hardware and dispatches from
+the most popular ML models, although we do not guarantee completeness. So for
+many users, the default specs will provide sufficient performance.
 
-Currently, the dispatch tuner that generates tuning specs is still experimental
-and hosted
-[in an external repo](https://github.com/nod-ai/shark-ai/tree/main/tuner). This
-document describes how to work with tuning specs generated by the SHARK Tuner
-or produced manually, but it does not go into detail on how to generate these
-specs.
+But for users looking to get the best performance on custom models and hardware
+targets, the IREE compiler also supports user-provided tuning specs. These
+specs can be made manually or generated using SHARK Tuner, which is still under
+development.
 
-## Flags
+!!! info
+    Additional information on installation and a hands-on
+    [example](https://github.com/nod-ai/shark-ai/tree/main/sharktuner/model_tuner)
+    is available in an external
+    [repository](https://github.com/nod-ai/shark-ai/tree/main/sharktuner).
```

Comment:
How about this rewording?

```suggestion
!!! info
    For more information about SHARK Tuner, see its source in the
    [shark-ai GitHub repository](https://github.com/nod-ai/shark-ai/tree/main/sharktuner)
    and the
    [Model Tuner example](https://github.com/nod-ai/shark-ai/tree/main/sharktuner/model_tuner).
```

* Lead with the general link (README, project overview) _then_ put the specific link (example package)
* https://developers.google.com/style/cross-references#descriptive-link-text

    > For the link text itself, use short, unique, descriptive phrases that provide context for the material that you're linking to.